src/hermezjs-typings.d.ts

```diff
@@ -780,7 +780,7 @@ declare module "@hermeznetwork/hermezjs/src/environment" {
 
   function isEnvironmentSupported(chainId: number): boolean;
 
-  // function getBatchExplorerUrl();
+  function getBatchExplorerUrl(): string;
```

Comment:
Why uncommenting this function? I think that you are not using it anywhere in this PR, right?

Reply:
It's used in `my-account.view` line 132